frontend/app/components/validation-date-input.js

```diff
@@ -0,0 +1,11 @@
+import Component from "@ember/component";
+
+export default Component.extend({
```

Comment:
Change this to a glimmer component!

Reply:
Changing it to a glimmer component breaks it. Is it necessary?

Reply:
I just took this into my own hands, hope you don't mind.
You are right, you're currently relying on methods with this component by the classic component.
In that situation, to not lose a lot of time I decided to change it to the class syntax and add the classic decorator. Best to update this once the addon gets updated